js/modules/k6/grpc/client.go

```diff
@@ -116,17 +117,27 @@ func (c *Client) Load(ctxPtr *context.Context, importPaths []string, filenames .
 		return nil, errors.New("load must be called in the init context")
 	}
 
-	f, err := protoparse.ResolveFilenames(importPaths, filenames...)
-	if err != nil {
-		return nil, err
+	initEnv := common.GetInitEnv(*ctxPtr)
+	if initEnv == nil {
+		return nil, errors.New("missing init environment")
+	}
+
+	// If no import paths are specified, use the current working directory
+	if len(importPaths) == 0 {
+		importPaths = append(importPaths, initEnv.CWD.Path)
 	}
 
 	parser := protoparse.Parser{
 		ImportPaths:      importPaths,
-		InferImportPaths: len(importPaths) == 0,
+		InferImportPaths: false,
+		Accessor: protoparse.FileAccessor(func(filename string) (io.ReadCloser, error) {
+			absFilePath := initEnv.GetAbsFilePath(filename)
+			// initEnv.Logger.Infof("gRPC trying to load %s from %s...", filename, absFilePath)
```

Comment:
Ah, I actually was supposed to delete this... It can be useful, but unfortunately it's going to be repeated for every initialized VU, so probably not ideal...

Reply:
Hm ... yeah, maybe we should embed it somewhere in the afero.Fs implementations ... :thinking: . But definetely not for this PR and I can't figure out how it should work/look so I am just going to drop it until someone needs it